deps/build.jl

```diff
@@ -0,0 +1,28 @@
+using BinDeps
+using Compat
+
+@BinDeps.setup
+
+libmysqlclient = library_dependency("libmysqlclient",
+                                    alias=["libmysqlclient.so",
```

Comment:
why does it need to extract under usr? that's where the source build is set up to install but binaries probably won't use that